src/DB.ts

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+import * as mongoose from 'mongoose';
+import * as Promise from 'bluebird';
+const label = '[fh-mongoose] ';
+
+const handlers = {
+  onError: function(err: Error) {
+    console.error(label, err.toString());
```

Comment:
Any good logger with typescript definitions? I think we should pick logger from beginning.
https://www.npmjs.com/package/@types/bunyan seems to be a good choice.